lexico/utils.py

```diff
@@ -3,15 +3,24 @@
 import os
 import json
 import sqlite3
-
 import arrow
 from wordnik import *
 from tabulate import tabulate
-
 from .errors import ConfigFileError
 
 API_URL = 'http://api.wordnik.com/v4'
 
+def incorrect_key(API_KEY):
+    check_url='http://api.wordnik.com/v4/word.json/cat/definitions?api_key={}'.format(API_KEY)
+    try:
+        if(swagger.urllib.request.urlopen(check_url).getcode()==200):
```

Comment:
Study how other API calls are being made.
Also, it is **never** a good idea to compound so many calls into a single statement.